### ISSUE TITLE
Issues building tests

### ISSUE THREAD

**Issue by pgdr** [Statoil/libres#3]
Linking error when building **libres** with tests.
```make
[ 16%] Linking C executable ../../bin/job_status_test
/usr/bin/ld: CMakeFiles/job_status_test.dir/job_status_test.c.o: undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
libjob_queue/tests/CMakeFiles/job_status_test.dir/build.make:97: recipe for target 'bin/job_status_test' failed
make[2]: *** [bin/job_status_test] Error 1
CMakeFiles/Makefile2:1946: recipe for target 'libjob_queue/tests/CMakeFiles/job_status_test.dir/all' failed
make[1]: *** [libjob_queue/tests/CMakeFiles/job_status_test.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by joakim-hove** [comment]
Hmmm - I thought we were all green; might be necessary to explicitly add: `target_link_library( ... pthread )`

**Comment by pgdr** [comment]
Sorry, forgot to add, this is Ubuntu 16.10!

**Comment by pgdr** [comment]
Solved in #11
